### ISSUE TITLE
Add support for negative numeric literals in meta examples

### ISSUE THREAD

**Issue by karlnapf** [shogun-toolbox/shogun#4464]
Currently, doing something as
```Real log_width = -1.0```
results in
```
...
    raise TypeError("Failed to tokenize input. Unknown text on line %d '%s'" % (t.lineno, t.value,))
TypeError: Failed to tokenize input. Unknown text on line 26 '-1.0
```

Something that should be relatively easy to add?

**Comment by karlnapf** [comment]
@sorig let me know if you can do that, otherwise Ill add it as entrance task

**Comment by sorig** [comment]
Fixed in #4469 

I have no idea how I missed that!
